src/CyBy/Draw/Internal/Graph.idr

```diff
@@ -703,6 +703,59 @@ mergeGraphs c (G o1 g) (G o2 t) =
     E e  => mergeGraphsOnBond (convert c) e g t
     None => mergeGraphs' g t
 
+-- Rotating of template around connecting atom (no merging!).
```

Comment:
I don't understand the "(no merging!)" part. Could you elaborate on that or remove it, if it is of no importance?